### DISCUSSION TITLE
Improved theming for Up/Down controls

### DISCUSSION THREAD

**Discussion by niksedk** [SubtitleEdit/subtitleedit#7106] [General]
I've tried to make new NumericUpDown controls for better theming:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/0745b247-6d6d-498d-b77c-e68c4bba754d)

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/420a3c42-77a2-45da-b631-5fa67a07a69f)

The new controls have mouse over/down color change. 
The existing arrow up/down keys should still work.

Give it a test and let me know if it works: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.13/SubtitleEditBeta.zip

---------

The current version looks like this with dark theme:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/936a7f48-9778-4a7a-ab9f-29669607ae33)

**Comment by jmaraujouy** [comment]
Nice job, @niksedk! This was one of the things I proposed in my #7072 "DarkTheme UI refresh?" topic.

In my opinion, now it is looking much better. 👍🏻

Thanks!

**Comment by jmaraujouy** [comment]
@niksedk

Have you noticed that, when you start SE, the up/down controls looks vertically stretched, but when you hover over them, they reset to their expected position/size?

Here's an example of what I mean:
![controls](https://github.com/SubtitleEdit/subtitleedit/assets/117078256/02393cb1-d096-4b01-a583-24d24396fb77)

**Comment by niksedk** [comment]
Yeah, hopefully fixed here: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.13/SubtitleEditBeta.zip

Also, any hint on the triangle layout? Shallow not filled? Smaller?

**Reply by jmaraujouy** [reply]
It's fixed now. Thank you, Nikolaj. 👍🏻👍🏻

<strike>About the triangle up/down icons, I like the _Windows 11_ style. They are a little smaller, and a little bit rounder.</strike>
**EDIT:** Sorry. I misread your first post; I thought the new buttons were actual images. So, probably what I proposed cannot be achieved.
